src/Compilers/CSharp/Test/Syntax/Parsing/RecordParsing.cs

```diff
@@ -2,6 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable disable
```

Comment:
curious why add nullable disable?

Reply:
> curious why add nullable disable?

Nullable analysis has zero value in tests. And add an extra const of suppressing warnings.